electron/logger.ts

```diff
@@ -0,0 +1,22 @@
+export function listenStdout(): Promise<string[]> {
```

Comment:
Please add a tsdoc or jsdoc comment and describe a little bit what the function does.
This will help other people like me to understand in the future, what the purpose of this function is.

Reply:
:+1: Sure